frontend/projects/dipa-frontend/src/app/modules/gantt/gantt-controls.service.ts

```diff
@@ -0,0 +1,10 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GanttControlsService {
+
+  fullScreen = false;
```

Comment:
Services in Angular (auch lokale Services die den UI-State speichern) sollten eigentlich immer Observables sein. Warum:
- Die Anwendung wird dadurch mehr "Reactive"
- Sollten wir später auf einen State-Store umstellen arbeiten die sowieso mit Observables
- Deine Componente muss nicht active nach Zustandsänderungen pollen. 
- i.d.R. nimmt man für lokale Zustände dann ein "BehaviourSubject" siehe z.B. https://medium.com/@luukgruijs/understanding-rxjs-behaviorsubject-replaysubject-and-asyncsubject-8cc061f1cfc0
- Für's subscriben bietet sich dann der "async" Operator an, dann muss man nicht aktiv unsubscriben.